### ISSUE TITLE
Production logging completely silenced?

### ISSUE THREAD

**Issue by cjlucani** [stve/silencer#20]
I'm stumped. This works beautifully on development but on production I'm getting no logging at all. Any clues as to why that might be happening?

Rails 4.1.2
Silencer 0.6.0

Application.rb

```
require File.expand_path('../boot', __FILE__)
require 'rails/all'
require 'silencer/logger'

Bundler.require(*Rails.groups)

module Fudge
  class Application < Rails::Application
    config.filter_parameters += [:latitude, :longitude]
    config.time_zone = 'Australia/Hobart'
    config.action_dispatch.perform_deep_munge = false
    config.assets.paths << Rails.root.join("vendor","assets","bower")
    config.assets.paths << Rails.root.join("public_assets")
    config.assets.paths << Rails.root.join("vendor")
    config.middleware.insert_before 0, "Rack::Cors", :debug => true, :logger => (-> { Rails.logger }) do
      allow do
        origins '*'
        resource '*',
          :headers => :any,
          :expose  => ['access-token', 'expiry', 'token-type', 'uid', 'client'],
          :methods => [:get, :post, :delete, :put, :options, :head]
      end
    end
  end
end
```

Production.rb

```
Rails.application.configure do
  config.cache_classes = true
  config.eager_load = true
  config.consider_all_requests_local       = false
  config.action_controller.perform_caching = true
  config.serve_static_assets = false
  config.assets.js_compressor = :uglifier
  config.assets.compile = false
  config.assets.digest = true
  config.action_mailer.raise_delivery_errors = true
  config.action_mailer.default_url_options = { host: '[hidden]' }
  config.action_mailer.delivery_method = :smtp
  config.action_mailer.smtp_settings = {
    ...
  }
  config.log_level = :info
  config.i18n.fallbacks = true
  config.active_support.deprecation = :notify
  config.log_formatter = ::Logger::Formatter.new
  config.active_record.dump_schema_after_migration = false
  config.middleware.swap Rails::Rack::Logger, Silencer::Logger, :silence => "/ping"
end
```

**Comment by stve** [comment]
Hi @cjlucani can you try using the latest release candidate of the gem `1.0.0.rc3`. It represents an extensive rewrite and the documentation in the README on GitHub reflects changes made in the 1.x versions.

**Comment by stve** [comment]
I've just released version `1.0.0`. Feel free to reopen should you continue to run into issues. Thanks!
